liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -119,7 +119,11 @@ private void generateFeatures() throws MojoExecutionException, PluginExecutionEx
         	
         	// In a multi-module build, generate-features will only be run on one project (the farthest downstream). 
         	// If this current project in the Maven Reactor is not that project or any of its upstream projects, skip it.  
-        	List<MavenProject> relevantProjects = getRelevantMultiModuleProjects(graph);
+        	boolean skipJars = true;
```

Comment:
Does "generate-features" do something meaningful today at LMP v3.10, in the Spring Boot JAR case?  Are we adding any value at all by complicating the path here?

Reply:
I doubt we have tested this scenario. So what would perform the most like it did before the changes went in to skip jar modules? Do we want to not skip jar modules at all for now for `generate-features`?

Reply:
I think that might cause problems since `generate-features` is called from dev mode... so in the non-springboot case, we could flag a conflict in `generate-features` (a jar and a war module without any downstream projects), that we didnt flag in the dev mode execution.  So the two need to behave the same in that regard. 

Since dev mode is not supported in the Springboot case, should we just add this SB check to the RunServerMojo and not for DevMojo and GenerateFeaturesMojo?